documentation/user_docs/docs/manual/Binary_operations.rst

```diff
@@ -2,110 +2,154 @@
 
-- ``w2_sqw_dnd`` is as above, i.e. an array of dnd-type sqw objects whose plot axes match element by element those of the array ``w1``.
+..
+   - ``w2_sqw_dnd`` is as above, i.e. an array of dnd-type sqw objects whose plot axes match element by element those of
+   the array ``w1``.
 
 - ``w2_dnd`` same rules as for sqw of dnd type above.
 
+- ``num`` is a numeric array following the size rules above.
+
 - ``scalar`` the same scalar is subtracted from every pix array in the array of sqw objects.
 
-There is one additional possibility
+dnd objects
+===========
 
-::
+Let us again take for our example the addition operator ``+``, and our initial ``dnd`` object is called ``w1`` .
 
-   wout = w1 + numeric_array
+.. note::
 
+   In the case of ``dnd`` objects, binary operations transform the ``s`` and ``e`` matrices
 
-- ``numeric_array`` is an array of scalars whose size matches the size of the sqw array. The output will continue to be an sqw of sqw type, with a pix array.
 
-dnd objects
------------
+Single dnd object
+-----------------
 
-Note that here we will use the phrase dnd object also to mean sqw object of dnd type, since the two are very closely related.
+You can add values to ``w1`` in the following ways:
 
-Single dnd object
-=================
+- ``w2_sqw`` is an ``sqw`` object a ``dnd`` (in ``data``) of identical size to ``w1``.
 
-The choices for a dnd object are:
+  ::
 
-::
+     wout = w1 + w2_sqw;
 
-   wout = w1 + w2_sqw
-   wout = w1 + w2_sqw_dnd_type
-   wout = w1 + w2_dnd
-   wout = w1 + scalar
+- ``w2_dnd`` is a ``dnd`` object commensurate with ``w1``.
 
+  ::
 
-The forms of these objects are as described for the sqw case. In all cases the output will be a dnd object. Note that the pix field of ``w2_sqw`` is ignored for this operation.
+     wout = w1 + w2_dnd;
 
 
+- ``num`` is a numeric array of the same size as the arrays of ``w1``.
 
-Array of dnd objects
-====================
+  ::
 
-As with arrays of sqw objects, there is one further choice compared to a single dnd:
+     wout = w1 + num;
 
-::
+- ``scalar`` is a single number, e.g. if you want to add 4.782 to all of the data in ``w1``.
+
+  ::
+
+     wout = w1 + scalar;
 
-   wout = w1 + numeric_array
 
+Array of dnd objects
+--------------------
+
+Similar to arrays of sqw objects.
 
-As for sqw objects, the numeric array has to be the same size as the array of dnd objects.
+As for sqw objects, arrays have to be the same size as the array of dnd objects with respectively commensurate array
+sizes or a scalar object as the same size of each.
 
 
 List of operations and their equivalent code
 --------------------------------------------
 
-The arithmetic operations above correspond to equivalent Matlab functions. You should never need to use these, but for reference the corresponding functions are:
+The arithmetic operations above correspond to equivalent Matlab functions. You should never need to use these, but for
+reference the corresponding functions are:
```

Comment:
Just a comment on MD formatting. We are encouraged to put frequent \n in our text to make separate lines for reviewing and working with the raw text. It would be good to have even shorter lines so that they fit within one half of a split-screen github comparison.